site/utils/exportUtils.ts

```diff
@@ -73,5 +73,25 @@ export const getExportSchema = (): Schema<ExportDisruptionData> => {
             type: String,
             value: (objectData: ExportDisruptionData) => objectData.severity,
         },
+        {
+            column: "Planned/Unplanned",
+            type: String,
+            value: (objectData: ExportDisruptionData) => objectData.disruptionType,
+        },
+        {
+            column: "Date added",
+            type: String,
+            value: (objectData: ExportDisruptionData) => objectData.creationTime,
```

Comment:
Old disruption don't have a creation time so when I export the data a lot of disruptions have N/A under "date added"

In the SIRI generator we have a function called `getDisruptionCreationTime` where if there is not a creation time we use the disruption history to get when the disruption was created. Can we reuse this function to populate the date added column?

Reply:
done